game-core/src/main/java/games/strategy/engine/framework/startup/ui/MetaSetupPanel.java

```diff
@@ -118,15 +116,6 @@ private void layoutComponents() {
     // top space
     add(new JPanel(), new GridBagConstraints(0, 100, 1, 1, 1, 1, GridBagConstraints.CENTER, GridBagConstraints.BOTH,
         new Insets(0, 0, 0, 0), 0, 0));
-
-    if (ClientSetting.SHOW_BETA_FEATURES.booleanValue()) {
```

Comment:
I don't think there's any other use for this setting, so removing it would be good.

Reply:
Looks like it can be removed. I'm a bit on the fence about this since I think we'll want this feature again in the medium term. Then again, YAGNI.

I think that means to remove it, but on its own branch so its easy to undo in case we want to use that code again. I'll follow up with another PR to do that.